exciting_environments/cart_pole/cart_pole_env.py

```diff
@@ -50,8 +50,8 @@ def __init__(
         physical_constraints: dict = None,
         action_constraints: dict = None,
         static_params: dict = None,
+        control_state: list = None,
```

Comment:
I am not 100% sure if I understand what the goal is with control state. We should discuss this.